### ISSUE TITLE
problem about odious.py

### ISSUE THREAD

**Issue by WangZZJJ** [Vegeta2020/SE-SSD#36]
I want use odiou_3D loss to my own network, but why the loss computed by odious.py can be -inf? Besides, the prediction will be NaN after training very little time?

**Comment by Vegeta2020** [comment]
It may be caused the unexpected values produced by the model without pre-training, if you used the pre-trained model, it would be avoided.

**Comment by Viczyf** [comment]
> It may be caused the unexpected values produced by the model without pre-training, if you used the pre-trained model, it would be avoided.

Have you thought about the reason why the loss can be -inf?

**Comment by Vegeta2020** [comment]
As the non-pre-trained model may produce unexpected values like negative ones, they cannot generate normal boxs for IoU calculation.
